charts/mattermost-team-edition/templates/route.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.route.enabled -}}
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  annotations:
+    haproxy.router.openshift.io/timeout: 4m
+    openshift.io/host.generated: "true"
+  labels:
+    app: {{ include "mattermost-team-edition.name" . }}
```

Comment:
the labels need to be similar as we have here: https://github.com/mattermost/mattermost-helm/blob/master/charts/mattermost-team-edition/templates/deployment.yaml#L6